worlds/sa2b/Items.py

```diff
@@ -107,12 +245,24 @@ def __init__(self, name, classification: ItemClassification, code: int = None, p
     **junk_table,
     **trap_table,
     **emeralds_table,
+    **eggs_table,
+    **fruits_table,
+    **seeds_table,
+    **hats_table,
+    **animals_table,
+    **chaos_drives_table,
     **event_table,
 }
 
 lookup_id_to_name: typing.Dict[int, str] = {data.code: item_name for item_name, data in item_table.items() if data.code}
 
-item_groups: typing.Dict[str, str] = {"Chaos Emeralds": [item_name for item_name, data in emeralds_table.items()]}
+item_groups: typing.Dict[str, str] = {
+    "Chaos Emeralds": [item_name for item_name, data in emeralds_table.items()],
```

Comment:
gut  feeling says list(emeralds_table), or list(emeralds_table.keys()) if you want to be explicit should be faster.